src/script/calling/CallingRepository.ts

```diff
@@ -1193,6 +1194,20 @@ export class CallingRepository {
     return this.mediaStreamQuery;
   };
 
+  private readonly updateActiveSpeakers = (wuser: number, conversationId: string, rawJson: string) => {
+    const call = this.findCall(conversationId);
+    if (!call) {
+      return;
+    }
+
+    const activeSpeakers = JSON.parse(rawJson);
```

Comment:
`rawJson` is guaranteed to be at least `{}`, right?

Reply:
avs said it's fine